core/include/grids/populator.hpp

```diff
@@ -8,10 +8,19 @@
 #pragma once
 
 #include "utils/indexing.hpp"
+#include "definitions/invalid_values.hpp"
+#include <vecmem/containers/device_vector.hpp>
+#include <vecmem/containers/jagged_device_vector.hpp>
```

Comment:
:thinking: It may be a good idea to include the "data types" directly as well.

```c++
#include <vecmem/containers/data/vector_view.hpp>
#include <vecmem/containers/data/jagged_vector_view.hpp>
```

They are currently implicitly included by the device vector types, but depending on what we do with the code in the future, it may not stay like that for ever. (Though it probably will...)

Reply:
Mkay